R/getRunStatus.R

```diff
@@ -85,7 +85,10 @@ getRunStatus <- function(mydir = dir(), sort = "nf", user = NULL) {
         modelstat <- c(modelstat, as.numeric(readGDX(gdx = filename, "o_modelstat", format = "simplest")))
         modelstat_iter <- c(modelstat_iter, as.numeric(readGDX(gdx = filename, "o_iterationNumber", format = "simplest")))
       }
-      if (length(modelstat) > 0) out[i, "modelstat"] <- modelstat[which.max(modelstat_iter)]
+      if (length(modelstat) > 0) {
+        fileInfo <- file.info(modelstatFiles[file.exists(modelstatFiles)])
+        out[i, "modelstat"] <- as.numeric(readGDX(gdx = rownames(fileInfo)[which(fileInfo[, "mtime"] == max(fileInfo[, "mtime"]))], "o_modelstat", format = "simplest"))
+      }
```

Comment:
Pretty dense line.  Maybe easier to process the next time somebody has to touch that code:
```
if (length(modelstat) > 0) {
  fileInfo <- file.info(modelstatFiles)
  latest_gdx <- rownames(fileInfo)[which.max(fileInfo$mtime)]
  out[i,"modelstat"] <- as.numeric(readGDX(gdx = latest_gdx, "o_modelstat"))
}
```

Reply:
Otherwise OK to merge.